Expecto/Expecto.Impl.fs

```diff
@@ -9,6 +10,20 @@ open Expecto.Logging.Message
 open Helpers
 open Mono.Cecil
 
+//! The other option is to use a dedicated activity source for Expecto instead of adding it to the config
```

Comment:
One option is to have Expecto have a dedicate ActivitySource, the other is to add it the the Expecto config and allow a user to pass it along.